ocs_ci/framework/conf/default_config.yaml

```diff
@@ -17,6 +17,7 @@ RUN:
   run_id: null  # this will be redefined in the execution
   kubeconfig_location: 'auth/kubeconfig' # relative from cluster_dir
   cli_params: {}  # this will be filled with CLI parameters data
+  regional_dr: False # avoid some checks and fixtures related to ODF
```

Comment:
I think this should be changed to : import_without_odf .

You on some places directly access this key without .get so if it's not defined here it will fail with keyError.